pkg/scheduler/core/scheduling_queue.go

```diff
@@ -485,6 +509,15 @@ func (p *PriorityQueue) WaitingPods() []*v1.Pod {
 	return result
 }
 
+// Close closes the priority queue and signals condition variables that may be
```

Comment:
Normally I would say that implementation details like condition variables should not be mentioned in an exported function like this.

Reply:
Done. Made it more detailed in interface.Close(). Here just keep the comments neat.